src/bootloader/simple_pg.md

```diff
@@ -87,7 +87,7 @@ pub usingnamespace switch (builtin.target.cpu.arch) {
 pub const page = @import("page.zig");
 ```
 
-これで `boot.zig` から x64 のページングに関する機能を参照できるようになりました。
+これで `boot.zig` から x86 のページングに関する機能を参照できるようになりました。
```

Comment:
Our target is x86-64. I believe x64 is correct.

Reply:
Ah, you are correct, I just never heard x86_64 being called x64 ... I thought you meant x86 just as umbrella term rather than distinction between 32 and 64 bit. ... But maybe x86_64 would be even better to avoid any confusion.

Reply:
I removed that commit again.